tests/cmake_integration/generated_global/.gitignore

```diff
@@ -0,0 +1,3 @@
+build
+cpp/build
+idl/build
```

Comment:
You can remove this file. `build` directories are already covered by the root `.gitignore`.